src/cr/cube/matrix/measure.py

```diff
@@ -1326,6 +1356,351 @@ def blocks(self):
         return ZscoreSubtotals.blocks(self._weighted_cube_counts, self._dimensions)
 
 
+# === MARGINALS ===
+
+
+class _BaseSecondOrderMarginal(object):
+    """Base class for all second-order measure objects."""
+
+    def __init__(self, dimensions, second_order_measures, cube_measures, orientation):
+        self._dimensions = dimensions
+        self._second_order_measures = second_order_measures
+        self._cube_measures = cube_measures
+        self._orientation = orientation
```

Comment:
After living with it for a while I found `second_order_measures` to be unnecessarily long-winded and in `_Strand` I changed it to just `measures` with good results. Maybe since this is new we do the same here and then we can get around to changing it elsewhere sometime later.